src/deluge/drivers/sd/sd.c

```diff
@@ -38,10 +38,11 @@ int32_t sddev_power_on(int32_t sd_port) {
 
 	/* ---- Wait for  SD Wake up ---- */
 	sddev_start_timer(100); /* wait 100ms */
-	while (sddev_check_timer() == SD_OK) {
-		/* wait */
-		routineForSD(); // By Rohan
-	}
+	                        //	while (sddev_check_timer() == SD_OK) {
+	                        //		/* wait */
+	                        //		routineForSD(); // By Rohan
+	                        //	}
+	yieldingRoutineForSD((RunCondition)sddev_check_timer);
```

Comment:
do we need to keep the old code?

Reply:
Yeah it's to make it easier to go back if we need to remove the scheduler for the 1.2 release

Reply:
This change isn't supposed to be in this PR, it belongs in the other one. Pulling it out